content/GStreamerDRPAI.md

```diff
@@ -31,13 +31,23 @@ The plugin also provides you with the following parameters:
 |-----------------------|---------------------|--------:|----------------------------------------------------------------------|
 | **model**             | String              |     --- | The name of the pre-trained model and the directory prefix.          |
 | **multithread**       | Boolean             |    true | Use a separate thread for object detection.                          |
+| **tracking**          | Boolean             |    true | Track detected objects based on their previous locations.            |
 | **log-detects**       | Boolean             |   false | Print detected objects in standard output.                           |
 | **show-fps**          | Boolean             |   false | Render frame rates of video and DRPAI at the corner of the video.    |
 | **stop-error**        | Boolean             |    true | Stop the gstreamer if kernel modules fail to open.                   |
 | **max-video-rate**    | Float [0.001 - 120] |     120 | Force maximum video frame rate using thread sleeps.                  |
 | **max-drpai-rate**    | Float [0 - 120]     |     120 | Force maximum DRPAI frame rate using thread sleeps.                  |
 | **smooth-video-rate** | Float [1 - 1000]    |       1 | Number of last video frame rates to average for a more smooth value. |
 | **smooth-drpai-rate** | Float [1 - 1000]    |       1 | Number of last DRPAI frame rates to average for a more smooth value. |
+| **smooth-bbox-rate**  | Float [1 - 1000]    |       1 | Number of last bounding-box updates to average. (requires tracking)  |
+| **track-seconds**     | Float [0.001 - 100] |       2 | Number of seconds to wait for a tracked undetected object to forget it. |
+| **track-iou-thresh**  | Float [0 - 1]       |    0.25 | The threshold of IOU for tracking bounding-boxes. (1=exact overlap, 0.1=loose overlap) |
```

Comment:
is this the only relevant parameter that allows for tracking? And is it between continous detections? What of the model isn't fast enough?

Reply:
`track-seconds` And `track-iou-thresh` are parameters for tracking.
I made them a variable so it could be tested and change on different models.